wandb/sdk/lib/redirect.py

```diff
@@ -620,13 +688,19 @@ def uninstall(self):
         if not self._installed:
             return
         self._installed = False
+
         self.src_wrapped_stream.flush()
-        time.sleep(1)
-        self._stopped.set()
+        time.sleep(0.1)
```

Comment:
If we really need to add sleeps let's add a comment describing why it's required and why we chose 0.1